python/cugraph/cugraph/gnn/graph_store.py

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph
+from cugraph.experimental import EXPERIMENTAL__PropertyGraph as PropertyGraph
+from cugraph.community.egonet import batched_ego_graphs
+
+
+class CuGraphStore:
+    """
+    A wrapper around a cuGraph Property Graph that
+    then adds functions to match the DGL GraphStorage API.
+    However, this class only return cuGraph types.  Conversion to other types,
+    like DGLGraph, needs to be done in DGL or other external packages.
+
+    homogeneous graphs, graphs with no attributes - use Property Graph
+    hetrogeneous graphs - use PropertyGraph
+    """
+
+    __G = None
+    __clinet = None
```

Comment:
I think best practice is to only use class variable for constants and set all the instance variables in `__init__`.

Reply:
updated